content/guides/models/track/log/log-summary.md

```diff
@@ -55,6 +71,81 @@ for i in range(10):
     wandb.log(log_dict)
 ```
 
-Here's what the resulting min and max summary values look like, in pinned columns in the sidebar on the Project Page workspace:
+## View summary metrics
+
+You can view summary values within a run's **Overview** page or within your project's **Run Table**.
+
+{{< tabpane text=true >}}
+{{% tab header="Run Overview" value="overview" %}}
+
+1. Navigate to the W&P App.
+2. Select the **Workspace** tab.
+3. From the list of runs, select the name of the run that logged the summary values.
+4. Click on the **Overview** tab.
+5. Scroll down to the **Summary** section to view the summary values.
+
+{{< img src="/images/track/customize_summary.png" alt="" >}}
+
+{{% /tab %}}
+{{% tab header="Run Table" value="run table" %}}
+
+1. Navigate to the W&P App.
+2. Select the **Runs** tab.
+3. Within the **Runs** table, you can view the summary values within the columns based on the name of the summary value.
+
+{{% /tab %}}
+
+{{% tab header="W&B Public API" value="api" %}}
+
+You can use the W&B Public API to fetch the summary values of a run. 
+
+For example, the following code example demonstrates one way to retrieve the summary values logged to a specific run using the W&B Public API and Pandas:
```

Comment:
```suggestion
This code example demonstrates one way to retrieve the summary values logged to a specific run using the W&B Public API and pandas:
```
Nit: https://pandas.pydata.org/ is lowercase `pandas`

Reply:
Good catch!